.github/workflows/deploy-enterprise-test.yml

```
@@ -14,113 +15,23 @@ jobs:
-            default_max_vcpus: 640
-            expanded_max_vcpus: 640
-            required_surplus: 0
-            security_environment: JPL-public
-            ami_id: /aws/service/ecs/optimized-ami/amazon-linux-2023/recommended/image_id
-
-          - environment: hyp3-its-live-test
-            domain: hyp3-its-live-test.asf.alaska.edu
-            template_bucket: cf-templates-3o5lnspmwmzg-us-west-2
-            image_tag: test
-            product_lifetime_in_days: 14
-            default_credits_per_user: 0
-            default_application_status: APPROVED
-            cost_profile: DEFAULT
-            job_files: >-
-              job_spec/AUTORIFT_ITS_LIVE.yml
-            instance_types: r7gd.2xlarge,r7gd.4xlarge,r7gd.8xlarge
-            default_max_vcpus: 640
-            expanded_max_vcpus: 640
-            required_surplus: 0
-            security_environment: JPL-public
-            ami_id: /aws/service/ecs/optimized-ami/amazon-linux-2023/arm64/recommended/image_id
-
-          - environment: hyp3-ak-fire-safe-test
-            domain: hyp3-ak-fire-safe-test.asf.alaska.edu
-            template_bucket: cf-templates-kc34l8cyitfn-us-west-2
-            image_tag: test
-            product_lifetime_in_days: 14
-            default_credits_per_user: 0
-            default_application_status: APPROVED
-            cost_profile: DEFAULT
-            job_files: >-
-              job_spec/AK_FIRE_SAFE.yml
-              job_spec/INSAR_ISCE_BURST.yml
-            instance_types: r6id.xlarge,r6id.2xlarge,r6id.4xlarge,r6id.8xlarge,r6idn.xlarge,r6idn.2xlarge,r6idn.4xlarge,r6idn.8xlarge
-            default_max_vcpus: 640
-            expanded_max_vcpus: 640
-            required_surplus: 0
-            security_environment: ASF
-            ami_id: /aws/service/ecs/optimized-ami/amazon-linux-2023/recommended/image_id
-
-          - environment: hyp3-pism-cloud-test
-            domain: pism-cloud-test.asf.alaska.edu
-            template_bucket: cf-templates-1qca1gr649g17-us-west-2
-            image_tag: test
-            product_lifetime_in_days: 14
-            default_credits_per_user: 0
-            default_application_status: APPROVED
-            cost_profile: DEFAULT
-            job_files: >-
-              job_spec/INSAR_ISCE_BURST.yml
-            instance_types: r6id.xlarge,r6id.2xlarge,r6id.4xlarge,r6id.8xlarge,r6idn.xlarge,r6idn.2xlarge,r6idn.4xlarge,r6idn.8xlarge
-            default_max_vcpus: 640
-            expanded_max_vcpus: 640
-            required_surplus: 0
-            security_environment: ASF
-            ami_id: /aws/service/ecs/optimized-ami/amazon-linux-2023/recommended/image_id
-
-          - environment: hyp3-volcsarvatory-test
-            domain: hyp3-volcsarvatory-test.asf.alaska.edu
-            template_bucket: cf-templates-1uzy61g1h6omf-us-west-2
-            image_tag: test
-            product_lifetime_in_days: 14
+          - environment: hyp3-enterprise-test # TODO create this env
```

Added this to the PR TODOs:
```suggestion
          - environment: hyp3-enterprise-test
```